features/xhr.yml

```diff
@@ -17,6 +17,7 @@ compat_features:
   - api.XMLHttpRequest
   - api.XMLHttpRequest.abort
   - api.XMLHttpRequest.abort_event
+  - api.XMLHttpRequest.authorization_removed_cross_origin
```

Comment:
Some more keys that might be relevant?

- api.XMLHttpRequest.getAllResponseHeaders.header_names_lowercase
- api.XMLHttpRequest.responseType
- api.XMLHttpRequest.responseType.arraybuffer_value
- api.XMLHttpRequest.responseType.blob_value
- api.XMLHttpRequest.responseType.document_value
- api.XMLHttpRequest.responseType.json_value
- api.XMLHttpRequest.responseURL
- api.XMLHttpRequest.send.body_URLSearchParams_type
- api.XMLHttpRequest.setAttributionReporting (this one probably belongs to Attribution Reporting)
- api.XMLHttpRequest.setPrivateToken (this one belongs elsewhere, likely?)